DEVELOPERS.md

```diff
@@ -68,6 +68,18 @@
          `Console` tab to view printed logs and errors, the `Element` tab to
          inspect the HTML and CSS, the `Network` tab for network calls, etc).
 
+#### Debugging Add-in: Alternate way
+If the above method to debug the add-in is not working, one can try the following steps:
+   1. Delete the `C:\Users\{username}\.office-addin-dev-certs` folder.
+   2. Run `npx office-addin-dev-certs install`.
+   3. Now run `npm run dev-server` to startup the server and `npm run start` 
```

Comment:
- Comma after `Now`.

Also, npm run start starts the server and also sideloads the add-in. You don't need to run `npm run dev-server`.

Reply:
Made the change